libraries/GCS_MAVLink/GCS_Common.cpp

```diff
@@ -2749,7 +2749,7 @@ void GCS_MAVLINK::send_home_position() const
     // get home position from origin
     Vector3f home_pos_neu_cm;
     if (!home.get_vector_from_origin_NEU(home_pos_neu_cm)) {
-        return;
+        home_pos_neu_cm.zero();
```

Comment:
should be nan, and probably for q and offsets too

Reply:
Q and InvalidOrigin is now NAN

Reply:
fixed